spec/order_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'order'
+
+describe Order do
+
+  describe "#order_meal" do
+    it "lets me select multiple dishes" do
+      expect(subject.order_meal("big mac", "fries")).to eq([ "big mac", "fries" ])
+    end
+    it "gives an error if the item isn't on the menu" do
+      expect{subject.order_meal("apple")}.to raise_error "that's not on the menu!"
```

Comment:
Nice test for error raising.  Do you know why the syntax has to be different here? I.e. Why the curly braces are needed?  Send over your answer on Slack, if you have one or would like to know the answer from me :)